assets/js/script.js

```diff
@@ -0,0 +1,208 @@
+// Assignment Code
+var generateBtn = document.querySelector("#generate");
+
+function generatePassword() {
+  const userCriteria = finalCriteria();
+
+  //Main Array
+  const choicesArray = getChoices(userCriteria);
+
+  //Final password
+  const password = [];
+
+  //Loop
+  for (let index = 0; index < userCriteria.length; index++) {
+    //pick a random character from the main array
+    let randomCharacter =
+      choicesArray[Math.floor(Math.random() * choicesArray.length)];
+
+    //Push the character to the password array
+    password.push(randomCharacter);
+  }
+
+  return password.join("");
+}
+
+//Get the password length from the user
+function getLength() {
```

Comment:
Although this works, it is best practice to use ES6 (arrow) functions. This would mean that these functions would have to be moved to above the function that it is called in as ES6 functions are not hoisted like function declarations.
```suggestion
const getLength = () => {
```

Reply:
Done